pyaedt/edb_core/hfss.py

```diff
@@ -1024,6 +1024,84 @@ def create_lumped_port_on_net(
                 return edges_pts
         return True
 
+    @pyaedt_function_handler()
+    def create_vertical_circuit_port_on_clipped_traces(self, nets=None, reference_net=None, user_defined_extent=None):
+        """Create an edge port on clipped signal traces.
+
+        Parameters
+        ----------
+        nets : list, optional
+            String of one net or EDB net or a list of multiple nets or EDB nets.
+
+        reference_net : str, Edb net.
+             Name or EDB reference net.
```

Comment:
```suggestion
        reference_net : str, Edb net.
             Name or EDB reference net. The default is ``None``.
```
I'm not sure why you keep resolving my suggestions without incorporating any of them. If I'm making bad suggestions, I'd like to understand why. My doc edit is intended to improve readability, which include inserting articles like these: a, an, the. Also, messages should be sentences and include the "is" helping verb where needed.